langchain/src/storage/tests/ioredis.int.test.ts

```diff
@@ -4,27 +4,57 @@ import { test } from "@jest/globals";
 import { Redis } from "ioredis";
 import { RedisByteStore } from "../ioredis.js";
 
-test("RedisByteStore", async () => {
-  const store = new RedisByteStore({
-    client: new Redis({}),
+describe("RedisByteStore", () => {
+  const client = new Redis({});
+
+  afterEach(async () => await client.flushall());
+
+  afterAll(() => client.disconnect());
+
+  test("RedisByteStore", async () => {
```

Comment:
This old tests is unchanged, except it's now inside this describe block & the client isn't initialized inside the test